test/e2e/antreapolicy_test.go

```diff
@@ -311,12 +311,72 @@ func testCNPDropEgress(t *testing.T) {
 	executeTests(t, testCase)
 }
 
+// testBaselineNamespaceIsolation tests that a CNP in the baseline Tier is able to enforce default namespace isolation,
+// which can be later overridden by developer K8s NetworkPolicies.
+func testCNPBaselineNamespaceIsolation(t *testing.T) {
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	nsExpOtherThanX := metav1.LabelSelectorRequirement{
+		Key:      "ns",
+		Operator: metav1.LabelSelectorOpNotIn,
+		Values:   []string{"x"},
+	}
```

Comment:
it seems that this could be an ANP as well, doesn't have to be a CNP?
no need to change it really, I just wanted to confirm with you

Reply:
Yes it works for both ACNP and ANP